### DISCUSSION TITLE
Update the list

### DISCUSSION THREAD

**Discussion by Zeyniaktass** [IBAX-io/go-ibax#1838] [Rewards]
Dear ibax administrators, can you please clarify this issue, not everyone is aware of what is going on. One of the tg admins says that the existing ranking list will be updated and the other says it will not be updated.  

 • my question is will the last 3 retweet points count or not?  

• Was the last 3 retweet events intentional?  If yes, why didn't you inform beforehand, what was your intention?  

 • Tag 3 friends under the tweet and get 200 points (each time) will these points be given?  

 • The people who registered 2 days ago got ahead of the people who have been dealing with it for 2 months because of the last retweet error, will you solve this grievance?

**Comment by Zeyniaktass** [comment]
Can any admin give a clear explanation about these issues?  Otherwise, I do not think that I will invest in the long term with these uncertainties, unless there is an environment where I can get proper information and answers to my questions.

**Reply by Abdullahoda** [reply]
Çok haklısın bir açıklama yok güncelleme yok kaç gün oldu tek bir hareket görmedik..

**Reply by Zeyniaktass** [reply]
Çok sorumsuz bir ekip galiba ben yatırım yapacaktım ama bu durumda zor malesef hakkımızı arayacağımız bir ortam yok cevap vermeye tenezzül etmiyorlar.
